test/openAPI/features/wallet_generate_link_auth_code.feature

```diff
@@ -0,0 +1,24 @@
+@method=POST @endpoint=/linked-authorization/link-auth-code
+Feature: The endpoint to validates the link-code and its expiry and generates the link auth code.
+
+  @smoke @unit @positive
+  Scenario: Successfully validates the link-code and its expiry and generates the link auth code smoke type test
+    Given Wants to validate the link-auth-code and generate the auth code
+    And The link code is generated before POST /linked-authorization/link-auth-code
```

Comment:
Why is that since the request body requires only transactionId and linkCode?

transactionId is hardcoded as in other tests, it should be updated after using some ready-to-use database.

Reply:
ready-to-use database strategy was discussed since there was no endpoint to get started transaction, but all the standardized workflows after the started transaction should be followed to ensure the transaction is in the current state that is ready to accept this /link-auth-code endpoint

Reply:
Ok, link-code transaction and authenticate before link-auth-code

Reply:
As per the #44 I can see we are generating link code, linking the transaction and authenticating, but we also need to do the consent (/linked-authorization/consent) before calling link-auth-code

Reply:
Moved to https://github.com/GovStackWorkingGroup/bb-identity/pull/44#discussion_r1243120354, so resolving